presentation/src/main/java/com/sopt/presentation/calendar/calendarTimePicker/CalendarTimePickerRoute.kt

```diff
@@ -0,0 +1,218 @@
+                    modifier = Modifier.weight(1f),
+                    style = typography.b4SemiBold,
+                    color = colors.gray900
+                )
+                CircularCheckbox(
+                    isChecked = isChecked,
+                    onCheckedChange = {
+                        isChecked = it
+                        showPicker = !it
+                        if (isChecked) {
+                            selectedStartHour = null
+                            selectedStartMinute = null
+                            selectedEndHour = null
+                            selectedEndMinute = null
+                        }
+                    }
+
+                )
+            }
+
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 32.dp, bottom = 9.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                NoostakSubHeaderText(stringResource(R.string.text_calendar_appointment_time_check), modifier = Modifier.weight(1f))
+                Box(
+                    modifier = Modifier.size(24.dp),
+                    contentAlignment = Alignment.Center
+                ) {
+                    Image(
+                        imageVector = ImageVector.vectorResource(
+                            id = if (isChecked) R.drawable.ic_calendar_off else R.drawable.ic_calendar_on
+                        ),
+                        contentDescription = null,
+                        modifier = Modifier.fillMaxSize()
+                    )
+                }
+            }
+
+            if (showPicker) {
+                NoostakTimePicker { startHour, startMinute, endHour, endMinute ->
+                    selectedStartHour = startHour
+                    selectedStartMinute = startMinute
+                    selectedEndHour = endHour
+                    selectedEndMinute = endMinute
+                }
+            }
+
+            Spacer(modifier = Modifier.weight(1f))
+
+            NoostakBottomButton(
+                text = stringResource(R.string.text_calendar_appointment_next),
+                onButtonClick = {
+                    val selectTime = if (isChecked) null else "${selectedStartHour?.toString()?.padStart(2, '0')}:${selectedStartMinute?.toString()?.padStart(2, '0')} ~ ${selectedEndHour?.toString()?.padStart(2, '0')}:${selectedEndMinute?.toString()?.padStart(2, '0')}"
+                    onButtonClick(
+                        appointmentName,
+                        category,
+                        time,
+                        isSingleDateMode,
+                        dates,
+                        selectTime ?: "null"
+                    )
+                },
+                deactivateColor = NoostakTheme.colors.gray500,
+                activateColor = NoostakTheme.colors.gray900,
+                modifier = Modifier.padding(bottom = 16.dp)
+            )
```

Comment:
P1: 이미 상단에 전체 패딩 16dp를 주고 있기 때문에 버튼 하단에 패딩을 또 주게 되면 총 아래에 패딩 32dp가 적용됩니다. 패딩 제거해주세요!!
그리고 해당 뷰 진입 시 버튼이 비활성화 되어야 합니다. isEnabled 추가해주세요
```suggestion
            NoostakBottomButton(
                text = stringResource(R.string.text_calendar_appointment_next),
                onButtonClick = {
                    val selectTime = if (isChecked) null else "${selectedStartHour?.toString()?.padStart(2, '0')}:${selectedStartMinute?.toString()?.padStart(2, '0')} ~ ${selectedEndHour?.toString()?.padStart(2, '0')}:${selectedEndMinute?.toString()?.padStart(2, '0')}"
                    onButtonClick(
                        appointmentName,
                        category,
                        time,
                        isSingleDateMode,
                        dates,
                        selectTime ?: "null"
                    )
                },
                deactivateColor = NoostakTheme.colors.gray500,
                activateColor = NoostakTheme.colors.gray900,
                isEnabled = {}
            )
```

Reply:
이거 활성화로 변경돼서 패딩만 제거할게여 !!!